problemsets-mat223/modules/module1.tex

```diff
@@ -479,7 +479,20 @@
 	Let $\vec x=\xhat-\yhat$, $\vec y=3\xhat-\yhat$, and $\vec r=2\xhat+2\yhat$. Is
 	$\vec r$ a linear combination of $\vec x$ and $\vec y$?
 
-	XXX Finish
+	Assume $\vec r$ is a linear combination of $\vec x$ and $\vec y$ for some unknown coefficients $\alpha_1$ and $\alpha_2$. Then
+	\begin{align*}
+	    \vec r&=\alpha_1\vec x+\alpha_2\vec y\\
+	   \implies 2\xhat+2\yhat&=\alpha_1(\xhat-\yhat)+\alpha_2(3\xhat-\yhat)\\
+	   \implies 2\xhat+2\yhat&=(\alpha_1+3\alpha_2)\xhat+(-\alpha_1-\alpha_2)\yhat.
+	\end{align*}
+	Thus, it is equivalent to solve the two-equations system
+	\begin{equation}
```

Comment:
You can use `\[...\]` for this. Only use `\begin{equation}` if you need a number equation to refer to later.